babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -874,6 +874,26 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testCreateTableDateFormatDateOnlyColumnLevelAttributes() {
+    final String sql = "create table foo (bar date)";
+    final String expected = "CREATE TABLE `FOO` (`BAR` DATE)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testCreateTableDateFormatWithFormatStringColumnLevelAttributes() {
+    final String sql = "create table foo (bar date format 'YYYY-MM-DD')";
+    final String expected = "CREATE TABLE `FOO` (`BAR` DATE FORMAT 'YYYY-MM-DD')";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testCreateTableDateFormatWithOtherColumnLevelAttributes() {
```

Comment:
Here also, I would suggest renaming this to something similar to:

testCreateTableWithDateFormatStringAndOtherColumnLevelAttributes()

Reply:
Updated in https://github.com/googleinterns/calcite/pull/103/commits/90fafc4decfc6b71ddaa415ff891a422d4bc4fa0